src/NotificationTemplateTranslation.php

```diff
@@ -380,18 +366,18 @@ public static function showAvailableTags($itemtype)
                 $allowed_values = '';
             }
 
-            if ($values['type'] == NotificationTarget::TAG_LANGUAGE) {
+            if ($values['type'] === NotificationTarget::TAG_LANGUAGE) {
                 $label = sprintf(__('%1$s: %2$s'), __('Label'), $values['label']);
             } else {
                 $label = $values['label'];
             }
             $rows[] = [
                 'values' => [
-                    ['content' => $tag],
-                    ['content' => $label],
-                    ['content' => $event],
-                    ['content' => $action],
-                    ['content' => $allowed_values],
+                    ['content' => htmlspecialchars($tag)],
+                    ['content' => htmlspecialchars($label)],
+                    ['content' => htmlspecialchars($event)],
+                    ['content' => htmlspecialchars($action)],
+                    ['content' => htmlspecialchars($allowed_values)],
```

Comment:
IMHO, values should be escaped by default. We should maybe add a `formatter` property to adapt the way the data is printed, as we do in `components/datatable.html.twig`.

Anyway, this can be done in another PR. Current usages are OK.